v0/destinations/hs/HSTransform-v2.js

```diff
@@ -351,17 +363,41 @@ const batchEvents = destEvents => {
           batchedResponse.destination
         )
       );
-    } else if (event.message.operation === "createContacts") {
+    } else if (event.message.source && event.message.source === "rETL") {
+      const { endpoint } = event.message;
+      maxBatchSize = endpoint.includes("contact")
+        ? MAX_BATCH_SIZE_CRM_CONTACT
+        : MAX_BATCH_SIZE_CRM_OBJECT;
+      if (operation) {
+        if (operation === "createObject") {
+          createAllObjectsEventChunk.push(event);
+        } else if (operation === "updateObject") {
+          updateAllObjectsEventChunk.push(event);
+        }
+      } else {
+        throw new CustomError("[HS]:: Error in getting operation", 400);
+      }
+    } else if (operation === "createContacts") {
       // Identify: making chunks for CRM create contact endpoint
       createContactEventsChunk.push(event);
-    } else if (event.message.operation === "updateContacts") {
+    } else if (operation === "updateContacts") {
       // Identify: making chunks for CRM update contact endpoint
       updateContactEventsChunk.push(event);
     } else {
       eventsChunk.push(event);
```

Comment:
This was kept for rETL source earlier. Do we need this section now? We can remove this and its linked code?

Reply:
removed the general parts.

Reply:
for update we have `updateContacts` and `updateObjects` keywords respectively